pkg/apis/eksctl.io/v1alpha5/vpc.go

```diff
@@ -58,6 +66,14 @@ const (
 	SubnetTopologyPrivate SubnetTopology = "Private"
 	// SubnetTopologyPublic represents publicly-routed subnets
 	SubnetTopologyPublic SubnetTopology = "Public"
+
+)
+
+var (
+	// True holds true value and can have it's address taken
+	True = true
+	// False holds false value and can have it's address taken
+	False = false
```

Comment:
usages of this should be replaced by `Disabled()`

Reply:
Done.